environment.go

```diff
@@ -949,6 +951,15 @@ func (e *envState) initTracer(ctx context.Context, useInternalDNS bool, tracer t
 	spanLimits := sdktrace.NewSpanLimits()
 	spanLimits.AttributeValueLengthLimit = OtelAttributeMaxLength
 
+	// Set up propagators. This allows traces from the UI to connect to traces from Evergreen.
+	propagatorB3 := b3.New(b3.WithInjectEncoding(b3.B3MultipleHeader | b3.B3SingleHeader))
+	otel.SetTextMapPropagator(
+		propagation.NewCompositeTextMapPropagator(
+			propagatorB3,
+			propagation.TraceContext{},
+			propagation.Baggage{}),
```

Comment:
do we need all these propagators? From [the docs](https://docs.honeycomb.io/send-data/javascript-browser/honeycomb-distribution/#distributed-trace-propagation) it looks like the front-end instrumentation is just using the trace context one.